datasets/aslg_pc12/aslg_pc12.py

```diff
@@ -0,0 +1,85 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""ASLG-PC12: Synthetic English-ASL Gloss Parallel Corpus 2012"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import datasets
+
+_DESCRIPTION = """\
+Synthetic English-ASL Gloss Parallel Corpus 2012
```

Comment:
Maybe you can provide more details in the description ?
That's what is showed in the dataset pages at https://huggingface.co/datasets